agent_tutorial/index.html

```
@@ -0,0 +1,527 @@
+</section>
+
+<!-- Writing an MCP Server -->
+<section>
+    <section>
+        <h3>Writing an MCP Server (Python)</h3>
+        <pre><code class="language-python">from mcp.server.fastmcp import FastMCP
+
+mcp = FastMCP("my-tools")
+
+@mcp.tool()
+def greet(name: str) -> str:
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)